src/briefcase/integrations/android_sdk.py

```diff
@@ -145,8 +152,12 @@ def avdmanager_path(self):
 
     @property
     def emulator_path(self):
-        emulator = "emulator.bat" if self.command.host_os == "Windows" else "emulator"
-        return self.root_path / "tools" / "bin" / emulator
+        emulator = "emulator.exe" if self.command.host_os == "Windows" else "emulator"
```

Comment:
Good catch. I confirm on my Windows VM that it's `emulator.exe`. I suspect you can just do `emulator` not `emulator.exe`, though.

Reply:
I think you're right; but explicit vs implicit, etc...